mc2/controllers/docker/tests/test_docker_controller.py

```diff
@@ -50,6 +51,11 @@ def docker_controller(with_envvars=True, with_labels=True, **kw):
 
     kw.setdefault("owner", models(User, is_active=just(True)))
     kw.setdefault("organization", models(Organization, slug=slug))
+
+    # Prevent Hypothesis from generating domains with invalid characters
+    domain_urls = text(string.ascii_letters + string.digits + '-.')
```

Comment:
Honestly, because I didn't see `letters`.

But domains are made up of ASCII letters and in some locales `letters` may be different from `ascii_letters`.